ios/Classes/TiMapView.h

```diff
@@ -30,6 +30,9 @@
     NSMutableArray *polygonProxies;
     NSMutableArray *circleProxies;
     NSMutableArray *polylineProxies;
+    
+    //selected annotation
+    MKAnnotationView<TiMapAnnotation> * selectedAnnotation;
```

Comment:
Make sure to also deallocate it during `dealloc` by using `RELEASE_TO_NIL(selectedAnnotation)`.